docs/topics/gradle.md

```diff
@@ -258,33 +258,27 @@ kotlin.stdlib.default.dependency=false
 
 The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing different Kotlin projects. 
 
-Add the corresponding dependencies on test libraries:
-
-* For `commonTest`, add the `kotlin-test-common` and `kotlin-test-annotations-common` dependencies.
-* For JVM targets, use `kotlin-test-junit` or `kotlin-test-testng` for the corresponding asserter implementation and annotations mapping.
-* For Kotlin/JS targets, add `kotlin-test-js` as a test dependency. 
+Add the dependency `kotlin-test` to the `commonTest` source set, and the Gradle plugin will infer the corresponding 
+test dependencies for each source set:
+* `kotlin-test-common` and `kotlin-test-annotations-common` for common source sets
+* `kotlin-test-junit`/`kotlin-test-junit-5`/`kotlin-test-testng` for JVM source sets
+* `kotlin-test-js` for Kotlin/JS source sets
 
 Kotlin/Native targets do not require additional test dependencies, and the `kotlin.test` API implementations are built-in.
 
+Additionaly, you can use the `kotlin-test` dependency in any shared or platform-specific source set.
+
+For Kotlin/JVM Gradle uses JUnit 4 by default; therefore, the `kotlin('test')` dependency resolves to
+the variant for JUnit 4, namely `kotlin-test-junit`:
```

Comment:
`the variant for` – is this part necessary? the sentence seems OK if we just drop these words.
Additionally, `variant` is a term that was never introduced in our docs.

Reply:
Should I fix it in [what's new](https://github.com/JetBrains/kotlin-web-site/pull/2192/files#r607194270) too?:)